pkg/cnb/create_builder.go

```diff
@@ -34,11 +34,17 @@ func (r *RemoteBuilderCreator) CreateBuilder(keychain authn.Keychain, buildpackR
 		return v1alpha1.BuilderRecord{}, err
 	}
 
-	builderBuilder, err := newBuilderBuilder(buildImage, lifecycleImage, stack, r.KpackVersion)
+	builderBldr, err := newBuilderBldr(lifecycleImage, r.KpackVersion)
 	if err != nil {
 		return v1alpha1.BuilderRecord{}, err
 	}
 
+	builderBldr.AddStack(
+		stack.Spec.Id,
+		buildImage,
+		[]string{},
```

Comment:
one day, we shall have mixins :)

Reply:
Maybe, we could remove this field until we support it. I was just playing with what it looks like.

Reply:
It threw me off a little bit when I saw it getting passed in and had to do a double take. I'm alright with keeping it around if we throw a tiny comment about why we are passing in an empty slice 🙂

Reply:
Mixins are the next thing to work on :)